File: Hospital.Server/Entities/Response/PendingOrderResponse.cs
namespace Hospital.Server.Entities.Response
{
    public class PendingOrderResponse
    {
        public string OrderType { get; set; } = string.Empty;
        public long OrderId { get; set; }
        public string OrderNumber { get; set; } = string.Empty;
        public string PatientName { get; set; } = string.Empty;
        public string PatientDpi { get; set; } = string.Empty;
        public string CreatedAt { get; set; } = string.Empty;
        public int ItemCount { get; set; }
        public decimal TotalAmount { get; set; }
        public int PaymentType { get; set; }
    }
}

File: hospital.client/src/types/PendingOrderResponse.ts
export interface PendingOrderResponse {
  orderType: string;
  orderId: number;
  orderNumber: string;
  patientName: string;
  patientDpi: string;
  createdAt: string;
  itemCount: number;
  totalAmount: number;
  paymentType: number;
}
